csrc/u8g2_polygon.c

```diff
@@ -70,7 +70,7 @@ static uint8_t pge_Next(struct pg_edge_struct *pge) PG_NOINLINE;
 static uint8_t pg_inc(pg_struct *pg, uint8_t i) PG_NOINLINE;
 static uint8_t pg_dec(pg_struct *pg, uint8_t i) PG_NOINLINE;
 static void pg_expand_min_y(pg_struct *pg, pg_word_t min_y, uint8_t pge_idx) PG_NOINLINE;
-static void pg_line_init(pg_struct * const pg, uint8_t pge_index) PG_NOINLINE;
+static void pg_line_init(pg_struct *pg, uint8_t pge_index) PG_NOINLINE;
```

Comment:
I got this error:

```
u8g2_polygon.c:248: error: (984) type redeclared
u8g2_polygon.c:248: error: (1098) conflicting declarations for variable "pg_line_init" (csrc/u8g2_polygon.c:247)
```

Because `const` was missing in the actual implementation. I removed const from the prototype because the methods above also don't have `const`.